pkg/kudoctl/packages/reader/read_dir.go

```diff
@@ -44,8 +45,18 @@ func ReadDir(fs afero.Fs, path string) (*packages.Package, error) {
 // FromDir walks the path provided and returns package files or an error
 func FromDir(fs afero.Fs, packagePath string) (*packages.Files, error) {
 	if packagePath == "" {
-		return nil, errors.New("path must be specified")
+		return nil, fmt.Errorf("path must be specified")
 	}
+
+	if !filepath.IsAbs(packagePath) {
+		// Normalize package path to provide more meaningful error messages
+		absPackagePath, err := filepath.Abs(packagePath)
+		if err != nil {
+			return nil, fmt.Errorf("failed to normalize package path %s: %w", packagePath, err)
```

Comment:
again I would just %v, I don't think we need to expose the file error types

Reply:
Done